### ISSUE TITLE
Devenv isTesting is not backwards compatible

### ISSUE THREAD

**Issue by shyim** [cachix/devenv#1279]
```
       … while calling the 'seq' builtin
         at «github:NixOS/nixpkgs/4c86138ce486d601d956a165e2f7a0fc029a03c1»/lib/modules.nix:322:18:
          321|         options = checked options;
          322|         config = checked (removeAttrs config [ "_module" ]);
             |                  ^
          323|         _module = checked (config._module);

       … while calling the 'throw' builtin
         at «github:NixOS/nixpkgs/4c86138ce486d601d956a165e2f7a0fc029a03c1»/lib/modules.nix:298:18:
          297|                     ''
          298|             else throw baseMsg
             |                  ^
          299|         else null;

       error:
       error: The option `devenv.isTesting' does not exist. Definition values:
       - In `<unknown-file>': false

✖ Command produced the following output:


Error:   × Command `/nix/store/rjvg7ah5x1im5w0rcmsjp9qbnxsbvv40-nix-2.21-devenv/bin/nix --show-trace --extra-experimental-features nix-command --extra-
  │ experimental-features flakes --option warn-dirty false --option eval-cache false --keep-going --max-jobs 6 eval .#devenv.processes --json`
  │ failed with with exit code 1
```

**Comment by shyim** [comment]
seems to be fixed with https://github.com/cachix/devenv/commit/5c5b90fc26856b86ce8ecd87eb76742fc8022e39

**Comment by domenkozar** [comment]
Fixed in 1.0.7
